site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,30 +6,95 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
 ---
 
-Extensions are small software programs that customize the browsing experience. They enable users to
-tailor Chrome functionality and behavior to individual needs or preferences. They are built on web
-technologies such as HTML, JavaScript, and CSS.
+This page provides a brief introduction to Chrome extensions and walks through the creation of a 
+"Hello, World!" extension.
 
-An extension must fulfill a [single purpose][1] that is narrowly defined and easy to understand. A
+
+## About extensions {: #intro }
+
+Extensions are small software programs that customize the browsing experience. They let users tailor
+Chrome functionality and behavior in many ways, providing things like:
+
+* Productivity tools
+* Web page content enrichment
+* Information aggregation
+* Fun and games
+
+These are just a few examples of the many things that extensions can do. See the [Chrome Web
+Store][cws] to see thousands of different examples of published extensions.
+
+### How do extensions work? {: #basics }
+
+Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
+sandboxed execution environment and interact with the Chrome browser.
+
+{# TODO: put a diagram here that is useful and helps with context #}
+{% if false %}
+
+Consider the context for this
+-- Chrome runs on a device, connecting the user to resources on the Web, as depicted in this context
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+{% endif %}
+
+Extensions let you "extend" the browser by using APIs to modify browser behavior and access web
+content. There are two aspects to consider:
+
+**The extensions user interface**&emsp;This provides a consistent way for users to manage their
+extensions.
```

Comment:
I'm a little confused by this section. On one hand I feel like it's trying to describe elements that the extension developer needs to consider, on the other the actual text seems to be about browser level considerations.

What's the objective of this section? Asking in order to see if I have any ideas on how we might be able to rephrase this to better address that goal.

Reply:
This is part of understanding what extensions are -- knowing the user experience is part of that. I'd like to get this merged, so can we steer away from optimization for now?

Reply:
I would prefer to gather further improvements in a follow-up ticket.

Reply:
That seems reasonable. I won't let good be the enemy of perfect here 😄